nlp_profiler/high_level_features/spelling_quality_check.py

```diff
@@ -3,8 +3,25 @@
 
 import pandas as pd
 from joblib import Memory
-from nltk.tokenize import word_tokenize
-from textblob import Word
+
+#alternative for textblob spell checker
+from symspellpy import SymSpell, Verbosity
+import pkg_resources
+
+# load a dictionary (this one consists of 82,765 English words)
+#there are other dictionaries as well look at https://www.kaggle.com/code/muhammetfaik/spellcheckingbenchmark
+#link to the used frequency dictionary https://raw.githubusercontent.com/wolfgarbe/SymSpell/master/SymSpell/frequency_dictionary_en_82_765.txt
+sym_spell = SymSpell(max_dictionary_edit_distance=2, prefix_length=7)
+dictionary_path = pkg_resources.resource_filename(
+    "symspellpy", "frequency_dictionary_en_82_765.txt"
+)
+# term_index: column of the term 
+# count_index: column of the term's frequency
+sym_spell.load_dictionary(dictionary_path, term_index=0, count_index=1)
+
+#importing fuzzy matching for scoring purposes
+from fuzzywuzzy import fuzz
```

Comment:
These two lines can be written as one line, comma-separated imports:
```python
from fuzzywuzzy import a, b
```

Reply:
done!